src/index.ts

```diff
@@ -4,13 +4,13 @@ import { Options, Result } from "./types";
 
 export async function githubAppJwt({
   id,
-  privateKey
-}: Options): Promise<Result> {
+  privateKey,
   // When creating a JSON Web Token, it sets the "issued at time" (iat) to 30s
   // in the past as we have seen people running situations where the GitHub API
   // claimed the iat would be in future. It turned out the clocks on the
   // different machine were not in sync.
-  const now = Math.floor(Date.now() / 1000) - 30;
+  now = Math.floor(Date.now() / 1000) - 30,
+}: Options): Promise<Result> {
   const expiration = now + 60 * 10; // JWT expiration time (10 minute maximum)
```

Comment:
can we move the `- 30` into the code? I think it's a safety margin that should be part of the code of `universal-github-app-jwt`

```suggestion
  now = Math.floor(Date.now() / 1000),
}: Options): Promise<Result> {
  const expiration = now + 60 * 10 - 30; // JWT expiration time (10 minute maximum) minus a 30 seconds safety margin
```

Reply:
Sure thing, that should make it a little less confusing to use, since passing `now` as `Date.now() / 1000` would be different than not passing `now` as using the default value.

Do you think the `iat` claim should have the 30 second margin as well? Or just the expiration?

Reply:
ah very good question! Yes, `iat` should have that margin, too

Reply:
I do not think the expiration part go the `-30` addition to limi thte overall time to 10 minutes in v1.1.0. Refer [v1.1.0 index.ts](https://github.com/gr2m/universal-github-app-jwt/blob/v1.1.0/src/index.ts#L15)